google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/prepare/base_entry_factory.py

```diff
@@ -63,6 +70,27 @@ def _format_id_with_hashing(
         return formatted_id[:cls.__ID_MAX_LENGTH - hash_length] + \
             hash.hexdigest()[:hash_length]
 
+    @classmethod
+    def _format_linked_resource(cls, linked_resource, normalize=True):
```

Comment:
```suggestion
    def _format_linked_resource(cls, linked_resource, normalize=False):
```
Since using the `normalize` flag might lead to broken resource links, especially if used in the BI connectors, I'd rather have it set to `False` by default to prevent misuse of this feature. WDYT?

Reply:
If the connector uses this method, the connectors need to be aware that broken links can be generated. When the linkedResource is normalized, otherwise it would be invalid and would throw an error on ingestion, so if the connector does not intent to have broken links, this method should not be called. They should not fill the linkedResource field instead.

Also If the normalization is skipped, the truncate logic still can generate broken resource links, so having it default to False, I don't think that it really adds benefit, since the main places that use it, will need to activate the normalize logic.

Reply:
Got it, makes sense. WDYT about adding such a warning in the doctring to prevent misuse?